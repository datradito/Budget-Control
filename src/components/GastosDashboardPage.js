import React from 'react';
import ListaGasto from './ListaGasto';
import FiltrosListaGasto from './FiltrosListaGasto';
import GastosSummary from './GastosSummary';

const GastosDashboardPage = () => (
    <div>
        <GastosSummary />
        <FiltrosListaGasto />
        <ListaGasto />
    </div>

);

export default GastosDashboardPage;