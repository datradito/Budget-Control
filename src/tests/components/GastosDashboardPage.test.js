import React from 'react';
import { shallow } from 'enzyme';
import GastosDashboardPage from '../../components/GastosDashboardPage';


test('Deberia render GastosDashboardPage', () => {
    const wrapper = shallow(<GastosDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});