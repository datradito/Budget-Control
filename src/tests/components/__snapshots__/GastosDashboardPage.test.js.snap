// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deberia render GastosDashboardPage 1`] = `
<div>
  <Connect(GastosSummary) />
  <Connect(FiltrosListaGasto) />
  <Connect(ListaGasto) />
</div>
`;
